src/include/duckdb/parser/parsed_data/table_data_info.hpp

```diff
@@ -0,0 +1,118 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/parser/parsed_data/table_data_info.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/parser/parsed_data/parse_info.hpp"
+#include "duckdb/common/types/data_chunk.hpp"
+#include "duckdb/common/vector.hpp"
+#include "duckdb/common/types/value.hpp"
+#include "duckdb/storage/table/column_data.hpp"
+
+namespace duckdb {
+
+struct TableDataInfo : public ParseInfo {
+public:
+	static constexpr const ParseInfoType TYPE = ParseInfoType::TABLE_DATA_INFO;
+
+	TableDataInfo() : ParseInfo(TYPE) {
+	}
+
+	TableDataType table_data_type;
+
+	//! Schema of the table
+	string schema;
+	//! Name of the table
+	string name;
+
+public:
+	unique_ptr<TableDataInfo> Copy();
```

Comment:
The same comment above applies to this and the other ParseInfo derived classes that are added